libcaliptra/src/caliptra_api.c

```diff
@@ -269,6 +269,9 @@ int caliptra_init_fuses(struct caliptra_fuses *fuses)
     if (!caliptra_ready_for_fuses())
         return NOT_READY_FOR_FUSES;
 
+    // Set Manuf service reg
+    caliptra_generic_and_fuse_write(GENERIC_AND_FUSE_REG_CPTRA_DBG_MANUF_SERVICE_REG, fuses->dbg_manuf_service);    
```

Comment:
I don't think this should be included with the fuses. If it doesn't exist, we could add a separate wrapper to write it.

Alternatively, there could be a function that takes the fuses and other init regs (dbg_manuf_service, wdt, trng config, pauser, etc.) that sets those and also wraps this fuse init call.

Reply:
done.